tests/cli.nix

```diff
@@ -8,7 +8,8 @@ runCommand "test-cli"
   {
     "kind": "Secret",
     "data": {
-      "foo": "bar"
+      "foo": "bar",
+      --json bar {}
```

Comment:
I appreciate that you want to contribute, but I don't think you fully understand what this file does.

This project uses a combination of Haskell, Nix and bash scripts inside the Nix files. Normally, I'd expect a contributor to be somewhat familiar with at least Nix and bash, as Nix users are this project's audience.

Based on your GitHub profile, it seems that you're learning a lot and that's admirable. Sadly, I'm not in a position to help you directly with that, but if you'd really like to contribute to `hercules-ci-agent`, you could study the technologies I've mentioned; at least bash, because it is a valuable skill for all sorts of automation.

Functional programming, including Haskell and Nix may be quite different from what you've learned so far. These languages solve problems that a starting software developer doesn't necessarily have to solve, such as the "devops" tasks that Nix is useful for. Haskell re-imagines how a programming language can work, making it a bit of an "outsider", which can be interesting or refreshing, but not something I would recommend early on.